### PR TITLE
Adding multiple new entries and editing Dreamstime information

### PR DESCRIPTION
**Pull request by marcoluzi**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -3459,8 +3469,8 @@
         "name": "Dreamstime",
         "url": "https://www.dreamstime.com/account/edit-profile",
         "difficulty": "easy",
-        "notes": "Click on the \"Delete account\" link towards the bottom of the page and then confirm your action in the opened pop-up.",
-        "notes_tr": "Sayfanın alt kısmına doğru gidin ve \"Hesabı Sil\" bağlantısını tıklayın ve ardından açılan pencerede işleminizi onaylayın.",
+        "notes": "Click on the \"Delete account\" link towards the bottom of the page and then confirm your action in the opened pop-up. Accounts can be deleted only if you have no activity registered (uploads, downloads, comments,blogs etc).",
+        "notes_tr": "Sayfanın alt kısmına doğru gidin ve \"Hesabı Sil\" bağlantısını tıklayın ve ardından açılan pencerede işleminizi onaylayın. Hesaplar yalnızca kayıtlı hiçbir etkinliğiniz yoksa (yüklemeler, indirmeler, yorumlar, bloglar vb.) silinebilir.",
```
Comment (tupaschoal) [inline _data/sites.json:3473]: Do you speak Turkish? If not we recommend entries of languages you do not speak to be deleted instead.

File: _data/sites.json [hunk 2]
```diff
@@ -4494,6 +4504,16 @@
         ]
     },
 
+    {
+        "name": "Franz",
+        "url": "https://meetfranz.com/user/profile",
+        "difficulty": "easy",
+        "notes": "Navigate to the bottom of the page where you have to option to delete your account.",
```
Comment (tupaschoal) [inline _data/sites.json:4511]:
```suggestion
        "notes": "Navigate to the bottom of the profile page where you have the option to delete your account.",
```

File: _data/sites.json [hunk 3]
```diff
@@ -4669,6 +4689,16 @@
         ]
     },
 
+    {
+        "name": "GAINSFIRE",
+        "url": "https://www.gainsfire.com/",
+        "difficulty": "medium",
```
Comment (tupaschoal) [inline _data/sites.json:4695]:
```suggestion
        "difficulty": "easy",
```
Given the description, this one sounds like it should be easy, no?

**Review by tupaschoal — COMMENTED** [review]
Thanks for the additions, just a few comments to iron out some questions and get it merged.

File: _data/sites.json (hunk 1 above)
Comment (marcoluzi) [reply _data/sites.json:3473]: @tupaschoal I don't. I used deepl to append the additional information.

Comment (tupaschoal) [reply _data/sites.json:3473]: Then can you please remove that line?

**Review by tupaschoal — APPROVED** [review]
Thanks for making the requested changes. As a note for future PRs, when you fix something pointed out on a comment/issue, mark the conversation as resolved, please :)

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**